types/react-loadable/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for react-loadable 3.3
+// Type definitions for react-loadable 4.0
 // Project: https://github.com/thejameskyle/react-loadable#readme
-// Definitions by: Diogo Franco <https://github.com/Kovensky>
+// Definitions by: Diogo Franco <https://github.com/Kovensky>, Oden S. <https://github.com/odensc>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.3
 
 import * as React from 'react';
```

Comment:
The `LoadedComponent` type itself can now just be an alias to `React.ComponentType`, which didn't exist at the time this was written.

Or we could take the opportunity to just remove it and use `React.ComponentType`; this update is a breaking change anyway.